content/c-sharp/concepts/methods/methods.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: 'Methods' 
+Description: 'Methods in C# are a block of code that can be called.'
+Subjects: 
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+CatalogContent: 
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+A **method** is a block of code used in [C#](https://www.codecademy.com/resources/docs/c-sharp) (and many other programming languages) containing statements. Each C# application has at least one method, called the 'Main', which is the starting point for the application. Methods are used to avoid the copy pasting of code, enabling better maintainability and readability of the code. Usually, a method has one or multiple dedicated purposes. 
+
+## Syntax
+
+To declare a function, the following parts must be present:
+
+* The access level (public, private, ...) and optional modifiers (abstract, sealed, ...)
+* The method return value (void, string, int, object, ...)
+* The method name
+* The (optional) method parameters
+
+All parts together form the **method signature**. Methods are written with a capital letter and camel case. 
+
+```pseudo
+// Declare a public method without return type and no parameters:
+public void PrintHappyHolidays()
+{
+	Console.WriteLine("Happy Holidays!");
+}
+
+// Declare a private method which returns an object and takes no parameters:
+private int CheckNumberOfCookies()
+{
+	return 7;
+}
+
+// Declare a public method which returns an object and takes an input parameter:
+public int CalculateLeftoverAmountOfCookies(int eaten)
+{
+	return CheckNumberOfCookies() - 7;
+}
+```
+
+
+## Example
```

Comment:
Formatting note here: Do not have more than one line break in a row, and headers should have a single line break before and after.